.sandbox/manifest.yaml

```diff
@@ -5,4 +5,4 @@ hooks:
 daemons:
   app:
     run:
-      cmd: rails s -b 0.0.0.0 -p 3000
+      cmd: PATH=$HOME/.rbenv/shims:"$PATH" rails s -b 0.0.0.0 -p 3000
```

Comment:
Please use `env`:

```
run:
  cmd: rails s ...
  env:
  - PATH=$HOME/.rbenv/shims:$PATH
```
Note: no "quotes"

Reply:
Updated this one and https://github.com/crafting-dev/demo-jobs-backend/pull/22. Could you help merge it? It requires write access for this repo.